### DISCUSSION TITLE
RefindPlus on Hackintosh PC

### DISCUSSION THREAD

**Discussion by atomizasser** [dakanji/RefindPlus#79] [General]
Hi.
I've a PC with windows and macOS with opencore. 
I've updated to the last version of refeindplus and now appears additional boot entry. With August version with default setting (attached) only opencore and windows entries were displayed. But now after update there is a new entry called "Boot MacOs from MyVol - Data" name. I've tried to remove it but I don't find any options to comments of dis comment to remove this entry. Please would be possible told me how remove this entry? Thanks
[config.conf.zip](https://github.com/dakanji/RefindPlus/files/7315288/config.conf.zip)
t

**Comment by dakanji** [comment]
You mean you have always had a Mac OS instance, "MyVol", and that this was not was not displayed before without you doing anything to stop the display? If that was the case, it meant there was a bug somewhere which has now been rectified.

Anyway, please raise this as a proper issue and provide the information requested in the issue template ... Log File, Screenshots etc.
